.github/workflows/translate.yml

```diff
@@ -0,0 +1,34 @@
+name: Translate Reports
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 8 * * *" # Run every day at 8 AM GMT
+
+jobs:
+  translate:
```

Comment:
I believe you will need to specify the environment for the job. e.g. from other workflow files, we use:
`environment: ${{ inputs.environment }}`

This is intended for production? 
In staging, can we configure the workflow to have the placeholders?

Reply:
(We can start by getting this migrated as-is first. :-))

Reply:
done!